src/main/target/CC3D/target.h

```diff
@@ -44,7 +44,7 @@
 
 #define USABLE_TIMER_CHANNEL_COUNT 12
 
-#define DEBUG_MPU_DATA_READY_INTERRUPT
+//#define DEBUG_MPU_DATA_READY_INTERRUPT
```

Comment:
Well it should not be removed. The default should be a comment. No idea why it is active in the cf master branch. Look at the other target.h files.